packages/app/cypress/e2e/runner/reporter.command_errors.cy.ts

```diff
@@ -0,0 +1,512 @@
+import * as specLoader from './support/spec-loader'
+import { createVerify } from './support/verify-failures'
+
+type VerifyFunc = (specTitle: string, verifyOptions: any) => void
+
+/**
+ * Navigates to desired error spec file within Cypress app and waits for completion.
+ * Returns scoped verify function to aid inner spec validation.
+ */
+function loadErrorSpec (options: specLoader.LoadSpecOptions): VerifyFunc {
+  const {
+    filePath,
+    hasPreferredIde = false,
+    mode,
+  } = options
+
+  specLoader.loadSpec(options)
+
+  // Return scoped verify function with spec options baked in
+  return createVerify({ fileName: Cypress._.last(filePath.split('/')), hasPreferredIde, mode })
+}
+
+describe('errors ui', {
+  viewportHeight: 768,
+  viewportWidth: 1024,
+  // Limiting tests kept in memory due to large memory cost
+  // of nested spec snapshots
+  numTestsKeptInMemory: 1,
```

Comment:
Might as well as just set this to 0.

Reply:
Done.

Reply:
The test file was still huge, so I like splitting it up even if we're not crashing chrome with `numTestsKeptInMemory: 0`.